i18n/miraheze/en.json

```diff
@@ -12,7 +12,8 @@
 	"miraheze-close-email-subject": "Miraheze Wiki ($1) Closed",
 	"miraheze-close-email-body": "Hello,\n\nUnfortunately, due to your wiki not having edits in the last 60 days, it has been closed due to the Dormancy policy. If you would still like to edit your wiki please go to Special:ManageWiki and uncheck the \"closed\" box to reopen it.\n\nPlease note that if you do not reopen your wiki it is possible that another user will adopt your wiki. If you do not reopen your wiki or no one adopts your wiki in 6 months starting from now, it could be eligible for deletion.",
 	"miraheze-donate": "Donate",
-	"miraheze-helpcenter": "Miraheze Help Centre",
+	"miraheze-helpcenter": "Miraheze Help Center",
+	"miraheze-faq": "FAQ",
```

Comment:
Should be alphabetized.

Reply:
@Universal-Omega Yeah, I can do that after, but for simplicity since I'm not very good with editing and synchronizing existing commits, would you be able to approve, and then I'll submit a separate PR to realphabetize? Thanks

Reply:
Editing the repo there will synchronize this PR https://github.com/dmehus/MirahezeMagic/tree/patch-1